android/src/com/mapswithme/maps/uber/UberInfo.java

```diff
@@ -0,0 +1,161 @@
+package com.mapswithme.maps.uber;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.Arrays;
+
+public class UberInfo implements Parcelable
+{
+  public static final Parcelable.Creator<UberInfo> CREATOR = new Parcelable.Creator<UberInfo>()
+  {
+
+    @Override
+    public UberInfo createFromParcel(Parcel source)
+    {
+      return new UberInfo(source);
+    }
+
+    @Override
+    public UberInfo[] newArray(int size)
+    {
+      return new UberInfo[0];
```

Comment:
Because we don't store the array of UberInfo object in to Parcel and I didn't pay attention on it, but I fixed it for future. Good point!